pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>co.cask.hydrator</groupId>
   <artifactId>kafka-plugins</artifactId>
-  <version>1.7.0-0.8.2.2-SNAPSHOT</version>
+  <version>1.7.0-SNAPSHOT-0.8.2.2</version>
```

Comment:
cdap determines if a version is a snapshot version if the suffix begins with 'snapshot'. So this will be seen as a snapshot, but the previous way would not.